test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -145,17 +149,64 @@ private static void initHelper(ClassWriter cw) {
 		mv.visitEnd();
 	}
 	
+	/*
+	 * This function should only be called in the 
+	 * TestWithFieldOnNonValueType test
+	 */
+	private static void testWithFieldOnNoneValueType(ClassWriter cw, String className, String[] fields) {
```

Comment:
should be NonValueType?

Reply:
yes, I've fixed it. 
  
I also fixed a couple other places where I made that mistake